gui/mass-remove.lua

```diff
@@ -1,13 +1,31 @@
 -- building/construction mass removal/suspension tool
 
+--@ module = true
+
+local toolbar_textures = (dfhack.textures.loadTileset('hack/data/art/mass_remove_toolbar.png', 8,12))
+
+local BASELINE_OFFSET = 42
+
 local gui = require('gui')
 local guidm = require('gui.dwarfmode')
 local utils = require('utils')
 local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
 
 local function noop()
 end
 
+local function get_l_offset(parent_rect)
+    local w = parent_rect.width
+    return BASELINE_OFFSET + (w+1)//2 - 34
+end
```

Comment:
since this is only used in one place, I suggest you inline it into `preUpdateLayout` so you can simplify the arithmetic